src/container/CategoryFilter.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { filterBooks } from '../actions/index';
+
+import CATEGORIES from '../constants';
+
+const CategoryFilter = ({ filterBooks }) => {
+  const handleFilterChange = event => filterBooks(event.target.value);
+
```

Comment:
- [ ] As is said in the requirements, the CategoryFilter component should be implemented as a presentational component, therefore, it is not aware of Redux and should be placed in `src/components` directory. You can read more about the difference between presentational and container components [here](https://redux.js.org/basics/usage-with-react#presentational-and-container-components).
![Screenshot from 2020-10-02 09-53-40](https://user-images.githubusercontent.com/30977845/94925179-2f971200-0495-11eb-9f32-371af5b5ce80.png)

- [ ] The `handleFilterChange` should be implemented by the BookList component and passed down to the CategoryFilter component as a prop
![Screenshot from 2020-10-02 09-57-07](https://user-images.githubusercontent.com/30977845/94925476-a7653c80-0495-11eb-862a-13b125e25b04.png)